src/pages/index.js

```diff
@@ -331,6 +334,18 @@ const Index = ({ data: initialData }) => {
     }
   }, [globalTabCount])
 
+  // set v4 beta cookie to false if meant to be on legacy
+  const v4BetaEnabled = usePrevious(user ? user.v4BetaEnabled : undefined)
```

Comment:
@amaliwanag2 Why the `usePrevious`?

Reply:
this was sometimes firing multiple times so I just did this defensively to only capture when it goes from undefined to true or false